test/es6/generators-functionality.js

```diff
@@ -1378,7 +1378,7 @@ var tests = [
             var iteratorReturningWithoutValue = CreateIterable(() => { return {done: false}; }, () => { return {done: true}; }, () => { return {done: true}; });
             gf = function* () { yield* iteratorReturningWithoutValue; }
             g = gf();
-            assert.areEqual({value: undefined, done: false}, g.next(), "As the value property is missing undefined is returned");
+            g.next();

```

Comment:
Right.  Because `g.next()` returns the `{done: false}` from the first arrow function passed to `CreateIterable`, correct?